mkdocs.yml

```diff
@@ -3,7 +3,9 @@ theme:
   name: material
   palette:
     primary: green
-use_directory_urls: false
 markdown_extensions:
   - footnotes
   - tables
+  - attr_list
```

Comment:
Required for attributes on links

Reply:
Hey @joshterrell805 - looks great. I will merge this.

Reply:
Thank you Leo! 🙏